README.md

```diff
@@ -127,7 +102,7 @@ $ env JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/ make  \
 
 ## Step 4: Build the benchmarks
 
-```bash
```

Comment:
I think we could do with some blurb here about "you will probably be adding your own benchmarks, but for the sake of example, we have some examples you can use".